copilot/fsd-pre-award-stores/manifest.yml

```diff
@@ -54,6 +54,7 @@ variables:
     from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-NotificationDeadLetterQueueURL
   ASSESSMENT_FRONTEND_HOST: "https://assessment.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
   SENTRY_TRACES_SAMPLE_RATE: 1.0
+  API_HOST: "http://fsd-pre-award-stores:8080"
```

Comment:
No `http://` here please, I believe 🙏

Reply:
It's deceptive because I think you've looked at eg `FUND_STORE_API_HOST`, which will have `http://` - however those are used for building URLs, while this needs to be a `host` in URI terms.

FUND_STORE_API_HOST should probably be called something like FUND_STORE_API_BASE_URL or something equally zany-but-slightly-more-specific